Library/Homebrew/blacklist.rb

```diff
@@ -83,5 +83,8 @@ def blacklisted?(name)
     A binary installer is available:
       https://www.haskell.org/platform/mac.html
     EOS
+  when "mysqldump-secure" then <<-EOS.undent
+    The creator of mysqldump-secure tried to game our popularity metrics.
```

Comment:
@cytopia Ok, I'm asking you directly: did you or did you not sign up new GitHub accounts for this purpose?